sdk/storage/Azure.Storage.Blobs/src/Sas/BlobSasBuilder.cs

```diff
@@ -349,6 +349,9 @@ public BlobSasQueryParameters ToSasQueryParameters(UserDelegationKey userDelegat
                 signedExpiry,
                 userDelegationKey.SignedService,
                 userDelegationKey.SignedVersion,
+                null, // authorized aad object id
```

Comment:
Should we add authorized aad object id, unauthorized add object id, and correlation id as settable properties on the `BlobSasBuilder`?

Reply:
These properties might be Data Lake only, in which case this comment doesn't make sense.